sample-input/benchmarks/c5g7/c5g7-cmfd.py

```diff
@@ -318,28 +319,29 @@
 track_generator.setNumThreads(num_threads)
 track_generator.generateTracks()
 
+
 ###############################################################################
-###########################   Running a Simulation   ##########################
+#                            Running a Simulation
 ###############################################################################
 
-solver = CPUSolver(geometry, track_generator)
-solver.setSourceConvergenceThreshold(tolerance)
+solver = CPUSolver(track_generator)
+solver.setConvergenceThreshold(tolerance)
 solver.setNumThreads(num_threads)
-solver.convergeSource(max_iters)
+solver.computeEigenvalue(max_iters)
```

Comment:
For what it's worth, this is a much better name than `convergeSource` was.  Does this adequately imply that a whole eigenvalue problem is being solved, not just computation of an eigenvalue for a single iteration, though?